project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_events.scss

```diff
@@ -59,3 +59,12 @@
   margin-bottom: 10px;
 }
 
+.node-event.view-mode-teaser {
```

Comment:
@guytepper 
I think this one can be nested under the previous `.node-event` class.